### ISSUE TITLE
http https detection

### ISSUE THREAD

**Issue by j0nk0** [davidfoerster/aptsources-cleanup#20]
my source.list contains multiple http and https entrys, eg:

```
deb http://http.kali.org/kali kali-rolling-only contrib main non-free
deb https://http.kali.org/kali kali-rolling-only contrib main non-free
```

these lines are not detected as duplicate

**Comment by davidfoerster** [comment]
They're not really equivalent so it’s hard to say if they’re actually duplicates. A host can serve different content for different protocols. Maybe I can introduce a new option for that. I’ll think about it.

Also, you don't need HTTPS to verify package integrity since Apt has its own certificate system.

**Comment by davidfoerster** [comment]
Fixed in f5417bdbb28243b115af0081efc34100069aee5f (v0.1.6.4).

### Usage

    sudo python3 -OEs aptsources-cleanup.zip --equivalent-schemes

Please refer to the command-line `--help` for details.

**Comment by davidfoerster** [comment]
I made the aforementioned feature the default in 14441fbe5ae395035727807dbac1e4e2d6df965c (v0.1.7.0).
